### PR TITLE
Multitenant holder implmentation

### PR DESCRIPTION
**Pull request by mattdean-digicatapult**
Implmentation of multitenant wallet for holder(s)

### CODE REVIEW

File: src/interface/hooks/use-post-multitenancy-wallet.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ */
+import { useCallback, useState } from 'react'
+import { useAuth0 } from '@auth0/auth0-react'
+
+import post from '../api/helpers/post'
+
+import { WALLET_TYPE } from '../../utils/env'
+import WALLET_STATUS from '../../utils/wallet-status'
+
+const mkWalletKey = () => {
```
Comment (n3op2) [inline src/interface/hooks/use-post-multitenancy-wallet.js:12]: what are the chances of getting same key?

File: src/interface/hooks/use-post-multitenancy-wallet.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        getAccessTokenSilently,
+        params,
+        body,
+        setStatus,
+        setError,
+        setStoreData,
+        () => WALLET_STATUS.READY
+      )
+    },
+    [getAccessTokenSilently]
```
Comment (n3op2) [inline src/interface/hooks/use-post-multitenancy-wallet.js:51]: so to my understanding this works out like a middleware?

**Review by n3op2 — DISMISSED** [review]

File: src/interface/hooks/use-post-multitenancy-wallet.js (hunk 1 above)
Comment (mattdean-digicatapult) [reply src/interface/hooks/use-post-multitenancy-wallet.js:12]: this is cryptographically secure so essentially zero. It's a 32 byte key so `1 in 115792089237316195423570985008687907853269984665640564039457584007913129639936`

File: src/interface/hooks/use-post-multitenancy-wallet.js (hunk 2 above)
Comment (mattdean-digicatapult) [reply src/interface/hooks/use-post-multitenancy-wallet.js:51]: depends what you mean by a middleware. I'd say it's a different model

**Review by jonmattgray — DISMISSED** [review]

**Review by n3op2 — APPROVED** [review]
Thanks for deleting unused hook

**Pull request merged**